Create code-of-conduct.it.md

GuglielmoPepe opened the pull request:
Italian translation version 1.4

GuglielmoPepe committed:
Create code-of-conduct.it.md

kalenobr commented on content/version/1/4/code-of-conduct.it.md:
```diff
@@ -0,0 +1,83 @@
+aliases = ["/version/1/4/it"]
++++
+
+# Codice di Comportamento dei Collaboratori
```
Gmail

kalenobr replied:
Block unkown gmails

sadeghrajabi replied:
tnx

sadeghrajabi commented:
just like

CoralineAda commented:
Hi @GuglielmoPepe, sorry I missed this. I just put out a call for other native speakers to review the translation.

Thank you so much for taking this on!

GuglielmoPepe commented:
Don't worry @CoralineAda , it's ok.

rikace commented on content/version/1/4/code-of-conduct.it.md:
```diff
@@ -0,0 +1,83 @@
+## Il nostro impegno
+
+Nell'interesse di promuovere un ambiente aperto ed accogliente, noi come
+collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro 
```
partecipazione del nostro

GuglielmoPepe replied:
I'm sorry @rikace , the comment is the correction or error?

rikace replied:
is the correction. It should be 
`collaboratori e responsabili ci impegnamo a rendere la partecipazione del nostro`

GuglielmoPepe replied:
Ok. I do not agree with you on this point: I think it is a "complemento di termine" so it is better   "collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro".

rikace replied:
you are right, however, 
"complemento di termine" ... shouldn't be "complemento del termine" ??

I don't see this part in the document

GuglielmoPepe replied:
Hi @rikace, sorry i missed this. The "complemento di termine" is https://it.wikipedia.org/wiki/Complemento_di_termine so it is better "collaboratori e responsabili ci impegnamo a rendere la partecipazione _**al nostro progetto e alla nostra comunità**_". I think.

rikace commented on content/version/1/4/code-of-conduct.it.md:
```diff
@@ -0,0 +1,83 @@
+
+Nell'interesse di promuovere un ambiente aperto ed accogliente, noi come
+collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro 
+progetto e alla nostra comunità un'esperienza libera da molestie per tutti, 
```
e della nostra comunità

GuglielmoPepe replied:
Ok. I do not agree with you on this point: I think it is a "complemento di termine" so it is better   "collaboratori e responsabili ci impegnamo a rendere la partecipazione al nostro progetto e alla nostra comunità".

rikace commented on content/version/1/4/code-of-conduct.it.md:
```diff
@@ -0,0 +1,83 @@
+personali o di ordine politico
+* Molestie pubbliche o private
+* Pubblicazione di informazioni private di un individuo, quali l'indirizzo 
+reale e/o elettronico, senza l'esplicito consenso
```
indirizzo fisico e/o elettronico

GuglielmoPepe replied:
Here I agree with you, better to use "indirizzo fisico e/o elettronico".

rikace reviewed (commented):
Beside these 3 small things, everything else looks good!

GuglielmoPepe committed:
Update code-of-conduct.it.md

Little correction "indirizzo reale e/o elettronico" -> "indirizzo  fisico e/o elettronico".
